framework/components/ecl-navigation/ecl-navigation-menus/_horizontal-menus.scss

```diff
@@ -11,8 +15,7 @@
   flex-grow: 1;
   list-style: none;
   margin: 0;
-  padding-left: 0;
-  transition: 0.2s;
```

Comment:
At the current snapshot, the background is also missing:

![aug 10 2017 5-12 pm](https://user-images.githubusercontent.com/1923476/29177569-4556d452-7def-11e7-904d-61253329b92c.gif)